docs/interpreter/spark.md

```diff
@@ -41,10 +41,52 @@ Spark Interpreter group, which consisted of 4 interpreters.
 </table>
 
 
+<br /><br />
+
+### Configuration
+<hr />
+
+Without any configuration, Spark interpreter works out of box in local mode. But if you want to connect to your Spark cluster, you'll need following two simple steps.
```

Comment:
I think it's important to point out that without setting SPARK_HOME it's running its included version of Spark? (that version depends on the profile used to build Zeppelin)